data_portal/viewsets.py

```diff
@@ -300,6 +334,41 @@ def get_queryset(self):
 
 class PresignedUrlViewSet(ViewSet):
 
+    def post(self, request):
```

Comment:
- Changing method name `post` to `create` will auto generate OpenAPI better. 
- Can you also update OpenAPI definition, pls. i.e. `make start` and, once up `make openapi` then `git diff swagger/swagger.json` for the diff.